automated/health-checks/health-check_RevPi_Connect_001.yaml

```diff
@@ -0,0 +1,41 @@
+device_type: RevPi_Connect
+job_name: RevPi_Connect - Standard health check 001 
+priority: medium
+visibility: public
+
+timeouts:
+  job:
+    minutes: 5
+  action:
+    minutes: 1
+  connection:
+    minutes: 2
+    
+actions:
+- deploy:
+    timeout:
+      minutes: 4
```

Comment:
If I understand correctly, the `deploy` action deploys the test scripts (among other things?) to the DUT. If this is the case then 4 minutes look overkill to me - 1 minute should be enough here.

What do you think? If the timeout of 4 minutes is adequate it should be documented.

Reply:
Copy & paste from old HealthCheck.

Reply:
I would still add a comment along the lines of

> Could possibly be lower; not tested

to the timeout.